x-pack/plugins/ml/common/types/anomaly_detection_jobs/model_snapshot.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { JobId } from './job';
+import { ModelSizeStats } from './job_stats';
+
+export interface ModelSnapshot {
+  job_id: JobId;
+  min_version: string;
+  timestamp: number;
+  description: string;
```

Comment:
Are all the fields required? Looks like `description` is optional

Reply:
According to online ML documentation this field is non-optional.